.github/workflows/gradle.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build Transformer
```

Comment:
You need a gradle wrapper validation

https://github.com/bndtools/bnd/blob/50e451283ce51cbe8366504bbe12915015b64d13/.github/workflows/cibuild.yml#L30-L31

Reply:
Done with latest